samples/test/talent.test.js

```diff
@@ -0,0 +1,141 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+const {assert} = require('chai');
+const cp = require('child_process');
+const {v4} = require('uuid');
+const {describe, it, before, after} = require('mocha');
+
+const talent = require('@google-cloud/talent').v4;
+
+const projectId = process.env.GCLOUD_PROJECT;
+const tenantService = new talent.TenantServiceClient();
+const companyService = new talent.CompanyServiceClient();
+const jobService = new talent.JobServiceClient();
+
+let tenant;
+let company;
+let job;
+
+let tenantId;
+let companyId;
+let jobId;
+
+const execSync = cmd => cp.execSync(cmd, {encoding: 'utf-8'});
+
+describe('Talent Solution Jobs API v4 samples', () => {
+  before(async () => {
+    const formattedParent = tenantService.projectPath(projectId);
+
+    [tenant] = await tenantService.createTenant({
+      parent: formattedParent,
+      tenant: {
+        externalId: v4(),
```

Comment:
perhaps we could make the `externalId`:

```
\`${Date.now()}-${v4()}\`
```

So that we can use the prefix to find old jobs and delete them -- unless there's a creation time on both resources?